beast/tools/verify_params.py

```diff
@@ -13,39 +13,43 @@
 from sys import exit
 
 
-def verify_range(param, param_name, param_lim):
+def verify_range(param, param_name, param_lim, noexit=False):
     # check if input param limits make sense
 	
     if any(p < param_lim[0] for p in param):
         print((param_name + ' min value not physical.'))
-        exit()
+        if not noexit: exit()
 
     if any(p > param_lim[1] for p in param):
         print((param_name + ' max value not physical.'))
-        exit()
+        if not noexit: exit()
 
-def check_grid(param, param_name, param_lim):
+def check_grid(param, param_name, param_lim, noexit=False):
     # check if input param limits and grid initialisation make sense
 
     param_min, param_max, param_step = param
 	
     if param_min < param_lim[0]:
         print((param_name + ' min value not physical.'))
-        exit()
+        if not noexit: exit()
 
     if param_max > param_lim[1]:
         print((param_name + ' max value not physical.'))
-        exit()
+        if not noexit: exit()
 
     if param_min > param_max:
         print((param_name + ' min value greater than max'))
-        exit()
+        if not noexit: exit()
 
     if (param_max-param_min) < param_step:
-        print((param_name + ' step value greater than (max-min)'))
-        exit()
-
-def verify_one_input_format(param, param_name, param_format, param_lim):
+        if param_max-param_min == 0.:
+            print('Warning: '+param_name+' grid is single-valued.')
```

Comment:
Nit picky point.  I don't think this is a warning.  There is no problem with single value parameters in a grid.  I worry that having it issue a warning makes it sound like this is bad.  Maybe something less "scary" like "Info"?

Reply:
Changed to "Note:"